### PR TITLE
[Task Seb] Update web3j version to 3.4  + Deploy the contract

### PR DESCRIPTION
**Pull request by sebaraba**
For this purpose I had to change:

- [x] Regenerate Contract using web3j 3.4.0 ( the latest version );
- [x] In `Application.java` --> fill in credentials of a wallet created using web3j command tools and deploy the contract;

screen shot 2018-05-31 at 21 11 16

### CODE REVIEW

File: src/main/java/org/web3j/sample/Application.java [hunk 1]
```diff
@@ -56,16 +56,16 @@ private void run() throws Exception {
         // We start by creating a new web3j instance to connect to remote nodes on the network.
         // Note: if using web3j Android, use Web3jFactory.build(...
         Web3j web3j = Web3j.build(new HttpService(
-                "https://rinkeby.infura.io/<your token>"));  // FIXME: Enter your Infura token here;
+                "https://rinkeby.infura.io/pmUdYLK9l52gdIsifNsc"));  // FIXME: Enter your Infura token here;
```
Comment (conor10) [inline src/main/java/org/web3j/sample/Application.java]: You need to reset the token and other credentials to being a placeholder

File: src/main/java/org/web3j/sample/Application.java [hunk 2]
```diff
@@ -56,16 +56,16 @@ private void run() throws Exception {
...
         log.info("Connected to Ethereum client version: "
                 + web3j.web3ClientVersion().send().getWeb3ClientVersion());
 
         // We then need to load our Ethereum wallet file
         // FIXME: Generate a new wallet file using the web3j command line tools https://docs.web3j.io/command_line.html
         Credentials credentials =
                 WalletUtils.loadCredentials(
-                        "<password>",
-                        "/path/to/<walletfile>");
+                        "seba12345",
```
Comment (conor10) [inline src/main/java/org/web3j/sample/Application.java]: ditto here

File: src/main/java/org/web3j/sample/Application.java [hunk 3]
```diff
@@ -46,6 +47,7 @@
 public class Application {
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
+    Scanner s = new Scanner(System.in);
```
Comment (conor10) [inline src/main/java/org/web3j/sample/Application.java]: Don't worry about this stuff, I'd rather that we leave out the scanner and don't request details from users, leaving placeholders in the code as was previously the case.

Comment (sebaraba) [reply src/main/java/org/web3j/sample/Application.java]: Ok, I updated this now. Sorry for the confusion!

Comment (conor10) [reply src/main/java/org/web3j/sample/Application.java]: One last change - you need to remove references to the Scanner on this line and in the import.

**Pull request merged**